packages/client/src/services/AssessmentService.js

```diff
@@ -14,16 +14,32 @@ export class AssessmentService {
     }
   }
 
+  static remove(assessment) {
+    try {
+      // Choose the correct method, url, and data to send
+      // in a request to the express packages/api/src/routes/assessment.js
+      // NOTE: the http.config file automatically adds /api to the front of your url
+      return Axios.delete(`http://localhost:4000/api/assessment/deleteAssessment/${assessment.id}`)
```

Comment:
The axios config in utils/http.config should allow you to just specify the route set in the backend

```suggestion
      return Axios.delete(`/assessment/deleteAssessment/${assessment.id}`)
```